modules/sandboxed-containers-deleting-namespace-web-console.adoc

```diff
@@ -4,24 +4,24 @@
 
 :_content-type: PROCEDURE
 [id="sandboxed-containers-deleting-namespace-web-console_{context}"]
-= Deleting a namespace using the web console
+= Deleting {sandboxed-containers-first} namespace
 
-You can delete a namespace by using the {product-title} web console.
```

Comment:
We can still delete the namespace from the web console. Maybe you want to keep that line and add " or the command-line." ?

Reply:
It was duplication as the next section is how to delete the namespace.

Reply:
There is a separate section for that. This is unnecessary duplication.